tests/specs/lemmas.k

```diff
@@ -9,6 +9,9 @@ module LEMMAS
     rule #lookup ( M:Map [ K1 <- V1 ] , K2 ) => #lookup ( K1 |-> V1 , K1 ) requires K1 ==Int  K2 [simplification]
     rule #lookup ( M:Map [ K1 <- _  ] , K2 ) => #lookup ( M         , K2 ) requires K1 =/=Int K2 [simplification]
 
+    rule 0 <=Int #lookup( _M:Map , _ )             => true                                       [simplification]
+    rule         #lookup( _M:Map , _ ) <Int pow256 => true                                       [simplification, smt-lemma]
```

Comment:
Why not `smt-lemma` on both?

Reply:
Because I only need it for the 2nd. Performance differs I guess.